src/other/protocols/importAsset.js

```diff
@@ -28,8 +28,7 @@ const getTypeFromMime = (mime, mimeTypes = MIME_TYPES) => {
 };
 
 const getTypeFromExtension = (name, extensionTypes = EXTENSION_TYPES) => {
-  const extension = path.extname(name);
-
+  const extension = toLower(path.extname(name).substring(1));
```

Comment:
Did this throw up a compile/linting issue, or was it just a bug you found and also fixed?

Reply:
Possibly a bug. I had a file with a .JPG extension that wasn't being recognized as an image asset; it was being handled as a network file.